src/api-client/src/modules/config.ts

```diff
@@ -21,10 +21,10 @@ export interface ApiReturn {
 export interface Config {
   applicationUrl: string;
   artifacts: Array<string>;
-  baseDir: string;
-  cwd: string;
+  baseDir?: string;
+  cwd?: string;
```

Comment:
I am annotating my config with this type, and I had type errors because these are defined as required.